<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QMainWindow" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма кредита</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_deposit_amount">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок / год</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_term">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_insert_rate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип кредита</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calc_credit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ежемесячный платеж:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Переплата по кредиту:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Общая выплата:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_monthly_payment">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_loan_overpayment">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_repayment">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_credit_type">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>121</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Аннуитетный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дифференцированный</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
